tests/test_ukrdict.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is really only needed if you use Python 2.

Reply:
poetry automatically create this file.
I remove this line and never be used in the python3 program.
Also, I'll remove such config from pre-commit
https://github.com/serhii73/ukrdict/blob/master/.pre-commit-config.yaml#L16

Reply:
poetry automatically create this file but maybe pre-commit created this line.
Sorry, but `scrapy` create 
```
> $ cat example.py                                                                            
# -*- coding: utf-8 -*-
import scrapy

```
such a line. 
[Scrapy](https://docs.scrapy.org) may be making a mistake?

Reply:
No. Scrapy is probably targeting dual python support which isn't needed for you as you declare py3-only requirement

Reply:
I got it. Thanks @webknjaz